src/main/java/javax/security/enterprise/authentication/mechanism/http/RememberMe.java

```diff
@@ -195,6 +194,14 @@
     String cookieName() default "JREMEMBERMEID";
     
     /**
+     * Flag to determine if remember me should be used.
+     * 
+     * @return Flag to determine if remember me should be used
+     */
+    @Nonbinding
+    boolean isRememberMe() default true;
+    
+    /**
      * EL expression to determine if remember me should be used. This is evaluated
      * for every request requiring authentication. The expression needs to evaluate
```

Comment:
Shouldn't the javadoc for isRememberMeExpression be updated to reflect that both immediate and deferred expression syntax is allowed?